Assets/Scripts/Client Scripts/UIManager.cs

```diff
@@ -166,7 +171,7 @@ string FormatGoal(Goal goal)
     {
         string tempGoal;
 
-        if (goal.goalState == Goal.GoalState.die) { tempGoal = "Kill "; } // Sets First word
+        if (goal.goalState == Goal.GoalState.Kill) { tempGoal = "Kill "; } // Sets First word
```

Comment:
Put this into constants folder and remove space. Make function where it takes verb + player and creates a sentence